### PR TITLE
refactor test_create_to_ical from test_encoding to test_examples.py

### PR DESCRIPTION
**Pull request by jacadzaca**
Refactor `test_create_to_ical`, `test_create_event_simple` and `test_unicode_parameter_name` from test_encoding along the lines of https://github.com/collective/icalendar/discussions/374, move them to `test_examples.py`.

### CODE REVIEW

File: src/icalendar/tests/test_examples.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+'''tests ensuring that *the* way of doing things works'''
```
Comment (niccokunzmann) [inline src/icalendar/tests/test_examples.py:1]: The docstring does not explain anything.

**Review by niccokunzmann — COMMENTED** [review]
I do not know what to do with it. I would need to reconnect to the value of refactoring this. It seems to be an ok refactoring. It can be merged as I have nothing against it except from the docstring. I mean in short: it does not hurt and I have nothing against it.

**Comment by jacadzaca** [comment]
I think creating a single file with tests that showcase practical icalendar usage is important, so as to expand the [examples section](https://icalendar.readthedocs.io/en/latest/usage.html#example).  We know that documentation is somewhat lacking in examples, because there have been issues where people ask how to do X, e.g. #317, #341, #317, #421, #425.

**Comment by niccokunzmann** [comment]
@jacadzaca, could you create an issue for this? I would find it nice if

1. The documentation with the examples shows up on readthedocs
2. That the code is actually running.

As a suggestion, one could use [doctest](https://docs.python.org/3/library/doctest.html)
I do think that this issue is a very good first-timer issue because of its repetitiveness, the simplicity and the range of skills you build up while contributing to it. I would really love to see it. If you do not like to make this issue, I will do it.

What do you think?

**Comment by jacadzaca** [comment]
I'm off to bed now, but I'll gladly file it tomorrow :)

**Comment by jacadzaca** [comment]
@niccokunzmann is #443 OK?

**Comment by niccokunzmann** [comment]
I commented. Could I edit it to make it more beginner friendly?

**Comment by niccokunzmann** [comment]
So, yes, thanks that is quite a good start. I could not have listed all the issues.

**Comment by jacadzaca** [comment]
Feel free to edit it :)

**Pull request merged**